api/incidents/incidentsRouter.js

```diff
@@ -456,16 +455,31 @@ router.get('/download', async (req, res) => {
       const range = createRange([start, today]);
       incidents = filterDataByDate(incidents, range);
     }
-
     let ids = req.query.ids || null;
 
     if (ids) {
-      let { ids } = queryString.parse(ids, { arrayFormat: 'comma' });
       incidents = incidents.filter((incident) => {
         return ids.indexOf(incident.id) > -1;
       });
     }
 
+    let rank = req.query.rank || null;
+    let ranks = [
```

Comment:
i think this is neccessary for the backend to have, so the FE can pull the ranks